template/docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Provide a narrative description of what you are trying to accomplish:
```

Comment:
Few things can be improved here:

 1. make everything a checkbox - makes it really easy for contributors, and we can later on parse it too, eventually
 2. add some text about the fact that they should check whether this goes to `develop`, `master` or a previous maintenance branch
 3. add a checkbox to make sure the tests pass
 4. add a checkbox to make sure they wrote documentation, if this introduces feature behavior
 5. add a checkbox to make sure the `CHANGELOG.md` was modified